pillar/nginx/chalkradio_redirect.sls

```diff
@@ -0,0 +1,34 @@
+{% set tlds = ['com', 'org', 'net'] %}
+
+nginx:
+    servers:
+    managed:
+      {% for tld in tlds %}
+      chalkradio-{{ tld }}-redirect:
+        enabled: True
+        config:
+          - server:
+              - server_name: chalkradiopodcast.{{ tld }}
+              - listen: 80
+              - listen: '443 ssl'
+              - listen: '[::]:80'
+              - listen: '[::]:443 ssl'
+              # The certificate uses subject alternative names.
+              - ssl_certificate: /etc/letsencrypt/live/amps-web.amps.ms.mit.edu/cert.pem
+              - ssl_certificate_key: /etc/letsencrypt/live/amps-web.amps.ms.mit.edu/privkey.pem
+              - ssl_stapling: 'on'
+              - ssl_stapling_verify: 'on'
+              - ssl_session_timeout: 1d
+              - ssl_session_tickets: 'off'
+              - ssl_protocols: 'TLSv1.2 TLSv1.3'
+              - ssl_ciphers: "TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_GCM_SHA256\
```

Comment:
If you want to split this across lines it's better to use the `>-` operator in YAML than the backslash escape.
For example:

```yaml
ssl_ciphers: >-
  TLS_AES_356_GCM_SHA384:
  TLS_CHACHA20_POLY1305_SHA256
```

Reply:
I just copied that from the original amps-redirect config, so we'd want to change that, too.

Reply:
I've changed the line wrapping in this configuration, but I'll do another PR outside of this one to fix all of the others in `pillar/nginx`.